python/lsst/ts/cRIOpy/M1M3TS/M1M3TSGUI.py

```diff
@@ -111,11 +114,11 @@ def setButtonsToState(self, state):
         state : `int`
             Current CSC state."""
         _bm = {
-            State.OFFLINE: [False, False, False, False, False],
-            State.STANDBY: [True, False, False, False, True],
-            State.DISABLED: [False, True, False, True, False],
-            State.ENABLED: [False, False, True, False, False],
-            State.FAULT: [False, False, False, True, False],
+            State.OFFLINE: [False, False, False, False, False, False],
```

Comment:
What are these entries? Please consider changing the lists to dicts, so that it is clearer what each entry means. If that is not practical, then a code comment explaining what each element is would be a big help.

Reply:
I started a common class for those state enabled/disabled stuff. Will simplify that code later.